packages/grid-pro/src/vaadin-grid-pro-edit-select.js

```diff
@@ -100,12 +100,17 @@ class GridProEditSelect extends Select {
           listBox.appendChild(item);
         });
 
-        // save the "keydown" event for Enter
-        listBox.addEventListener('keydown', (e) => {
-          if (e.keyCode === 13) {
-            this._enterKeydown = e;
-          }
-        });
+        // Save the "keydown" event for Enter to handle it on value change.
+        // Use capture phase because the item calls `preventDefault()`.
+        listBox.addEventListener(
+          'keydown',
+          (e) => {
+            if (e.keyCode === 13) {
+              this._enterKeydown = e;
+            }
+          },
+          true
```

Comment:
I see that tests fail if I remove `true`, although it is not that clear how it is that `preventDefault()` stops the propagation of the `keydown` event... is there, in fact, any custom logic that stops the event's propagation?

Reply:
Good point. I will investigate this a bit more to see if there is a cleaner solution.

Reply:
Pushed a fix which turned out to be a bit bigger than I expected, but hopefully now it makes more sense. The root cause of having to use capture phase is now described in a comment in `vaadin-select` where it is used.